src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -235,11 +235,29 @@ protected void wrapUp(GHEventInfo[] page) {
     }
 
     /**
+     * {@inheritDoc}
+     * @deprecated Beware that this lists only <em>public</em> repositories. Probably you wanted to use {@link #listRepositories(int)} instead.
+     */
+    @Deprecated
+    @Override
+    public synchronized Map<String, GHRepository> getRepositories() throws IOException {
+        return super.getRepositories();
+    }
+
+    /**
+     * {@inheritDoc}
+     * @deprecated Beware that this lists only <em>public</em> repositories. Probably you wanted to use {@link #listRepositories(int)} instead.
+     */
+    @Deprecated
+    @Override
+    public PagedIterable<GHRepository> listRepositories() {
```

Comment:
> This version is a short-cut for `listRepositories(int)`.

Well that is what you would expect, of course! But that is not at all what it does. The `super` implementation is in `GHPerson` and calls a totally different GitHub API with different behavior. Hence this PR.